src/modules/commander/PreflightCheck.cpp

```diff
@@ -335,7 +335,7 @@ static bool gyroCheck(orb_advert_t *mavlink_log_pub, unsigned instance, bool opt
 
 	if (ret) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: GYRO #%u UNCALIBRATED", instance);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: GYRO #%u UNCALIBRATED.", instance);
```

Comment:
Change no longer needed. Text already improved in this more recent PR:
https://github.com/PX4/Firmware/pull/10586/files#diff-a2bcb238e25d8a97ab2aee48f54dbc48R326